### ISSUE TITLE
Seeing `AnnotationQueryRunner failed URI malformed` failure when accessing the results

### ISSUE THREAD

**Issue by crazoes** [kernelci/kernelci-pipeline#699]

**Comment by crazoes** [comment]
@nuclearcat can you please suggest who would be the right person to fix this problem

**Comment by helen-fornazier** [comment]
@crazoes it is me, could you give me instructions how you reproduce it? It is just a miss configuration in the links in grafana, I can quickly fix that once I know how to reproduce

**Comment by crazoes** [comment]
@helen-fornazier you can try to check the following link :- 
https://fresh-kcidb-grafana-m6io3uhhiq-uc.a.run.app/d/build/build?orgId=1&var-datasource=default&var-origin=%27%%27&var-build_architecture=arm64&var-build_config_name=cros://chromeos-5.4/arm64/chromiumos-qualcomm.flavour.config&var-id=maestro:66969c63aa07c494f8d22ad8

**Comment by crazoes** [comment]
@helen-fornazier to reproduce this issue, you can try to access any kind of link in the `id` section which shows the details about the results. It will give you the error I mentioned in the description.

**Comment by nuclearcat** [comment]
@crazoes on grafana dashboard related matters it is mostly @helen-fornazier on grafana configuration, and kcidb is under management of Nikolai @spbnick .

**Comment by helen-fornazier** [comment]
@crazoes yes, the link is malformed, but the issue is in the place that redirects you to it.
which place more exactly are you clicking? Please share me the link with the filter you are using, since when I click on the  ids All builds it is working for me

**Comment by crazoes** [comment]
@helen-fornazier I think I found the root cause of this issue. It kind of happens now as well. When we click on the link to see the logs, the default value for the time is 100 years there, which is why we see this failure.

Can you please try to fix this?


![Image](https://github.com/user-attachments/assets/b2c87978-6fab-4a3b-a334-c566a187ca9a)

Check the time in top right corner

**Comment by helen-fornazier** [comment]
Just for the record: discussing with @crazoes we managed to figure that the bug was caused when origin was set to All. This problem is fixed now. @crazoes please check and close this issue after your confirmation.

**Comment by crazoes** [comment]
I can confirm this issue has been fixed now so closing this. Thanks @helen-fornazier :)
